pipeline/extract/gog/extract_gog.py

```diff
@@ -82,10 +82,10 @@ def get_price(game_dict: dict) -> float:
 
 def get_rating(game_dict: dict) -> float:
```

Comment:
This edited function looks good to me; well-edited to account for the new rating system